packs/typescript/NOTICE

```diff
@@ -0,0 +1,9 @@
+MIT License:
+
+Copyright (C) 2011-2015 Heroku, Inc.
```

Comment:
did this code actually come from Heroku or did you create the `detect` script on your own? I don't see a typescript buildpack... https://github.com/heroku/heroku-buildpack-typescript is a 404 for me.